### PR TITLE
Adding more logging to helm functions

### PR DESCRIPTION
**Pull request by jonahjon**
Signed-off-by: jonahjon <jonahjones094@gmail.com>

Adding logging to helm functions.

Moving the driver instantiation outside of the loop to not re-create multiple times.

### CODE REVIEW

**Review by abhay-krishna — APPROVED** [review]
/approve

File: release/pkg/operations/upload.go [hunk 1]
```diff
@@ -88,11 +94,7 @@ func UploadArtifacts(r *releasetypes.ReleaseConfig, eksArtifacts map[string][]re
 					// Trim -helm on the packages helm chart, but don't need to trim tinkerbell chart since the AssetName is the same as the repoName
 					trimmedAsset := strings.TrimSuffix(artifact.Image.AssetName, "-helm")
 
-					helmDriver, err := helm.NewHelm()
-					if err != nil {
-						return fmt.Errorf("creating helm client: %v", err)
-					}
-
+					fmt.Printf("Modifying helm chart for %s\n", trimmedAsset)
```
Comment (TerryHowe) [inline release/pkg/operations/upload.go:97]: No HelmLog here?

**Comment by TerryHowe** [comment]
test borked

File: release/pkg/operations/upload.go (hunk 1 above)
Comment (jonahjon) [reply release/pkg/operations/upload.go:96]: It's in a different dir, so didn't want to import it in.

File: release/pkg/helm/helm.go [hunk 2]
```diff
@@ -74,6 +74,8 @@ func GetHelmDest(d *helmDriver, r *releasetypes.ReleaseConfig, ReleaseImageURI,
 	}
 
 	helmChart := strings.Split(ReleaseImageURI, ":")
+	HelmLog.Info("Starting to modifying helm chart %s", helmChart[1])
```
Comment (abhay-krishna) [inline release/pkg/helm/helm.go:77]: I feel this should be helmChart[0] with the URI
```suggestion
	HelmLog.Info("Starting to modifying helm chart %s", helmChart[0])
```

**Review by abhay-krishna — APPROVED** [review]
/approve

**Pull request merged**